test/integration/globals/app-nav.js

```diff
@@ -866,6 +958,60 @@ context('App Nav', function() {
     cy.clock().invoke('restore');
   });
 
+  specify('add patient - clinician in one workspace', function() {
```

Comment:
I'm surprised this wasn't tested before..  I would have thought this worked the same when it was groups..

Reply:
Me too. It was getting covered before somehow... but I couldn't find where or how.